cmd/otelcontribcol/exporters_test.go

```diff
@@ -265,6 +266,22 @@ func TestDefaultExporters(t *testing.T) {
 			},
 			skipLifecycle: true,
 		},
+		{
+			exporter: "alertmanager",
+			getConfigFn: func() component.Config {
+				cfg := expFactories["alertmanager"].CreateDefaultConfig().(*alertmanagerexporter.Config)
+				cfg.HTTPClientSettings = confighttp.HTTPClientSettings{
+					Endpoint: "http://" + endpoint,
+				}
+				cfg.GeneratorURL = "opentelemetry-collector"
+				cfg.DefaultSeverity = "info"
+				// disable queue/retry to validate passing the test data synchronously
+				cfg.QueueSettings.Enabled = false
+				cfg.RetrySettings.Enabled = false
+				return cfg
+			},
+			expectConsumeErr: true,
+		},
```

Comment:
This lifecycle test should be added to your `metadata.yaml` file instead of explicitly in the builder config. mdatagen will then automatically generate the configuration in the component. See[ the schema doc](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/4ad75b45ff83472040b9f2c4636237a2b4b9876d/cmd/mdatagen/metadata-schema.yaml#L109) for reference.

I'm going to mark this as a requirement to merge the PR so that we do not create more work for ourselves in the future. @atoulme is doing spearheading a lot of work to deprecate this test file by removing lifecycle tests to components.

Reply:
Issue ref https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27849

Reply:
Looks like this will be handled in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30389/files

Reply:
#30389 is merged. This bit of code can be removed once you rebase from main.

Reply:
@sokoide do you have the requisite permission to review PRs in the repository? I understand they won't be "green check mark" but even "grey check mark" helps. If you don't have permission for "grey check mark" review, then that may be an issue we need to raise.

Reply:
@bryan-aguilar Thanks for your review. I updated the test config in metadata.yml under alertmanager exporter component. When removing above snippet from exporters_test, the unittest check failed. 
Kindly suggest if any further changes are required, thanks!